### PR TITLE
Add new tokens

### PR DESCRIPTION
**Pull request by leohar**
### Acala
<details>
  <summary>PARA</summary>

image
image

</details>

### Astar
<details>
  <summary>DOT</summary>

image

</details>

### Bifrost
<details>
  <summary>MOVR</summary>

[ED Source](https://github.com/bifrost-finance/bifrost/blob/develop/runtime/bifrost-kusama/src/lib.rs#L1340)
ED => 1 * micro(CurrencyId::Token(TokenSymbol::MOVR)),	// MOVR has a decimals of 10e18

image

</details>
<details>
  <summary>USDT</summary>

image
image

</details><details>
  <summary>PHA</summary>

[ED Source](https://github.com/bifrost-finance/bifrost/blob/develop/runtime/bifrost-kusama/src/lib.rs#L1340)
ED => 4 * cent(CurrencyId::Token(TokenSymbol::PHA)),	// 0.04 PHA, PHA has a decimals of 10e12

image

</details><details>
  <summary>vKSM</summary>

[ED Source](https://github.com/bifrost-finance/bifrost/blob/develop/runtime/bifrost-kusama/src/lib.rs#L1340)
ED => 10 * millicent(RelayCurrencyId::get())

image

</details>

### Karura
<details>
  <summary>GENS</summary>

image
image

</details>
<details>
  <summary>EQD</summary>

image
image

</details>
<details>
  <summary>CRAB</summary>

image
image

</details>

### CODE REVIEW

File: chains/v3/chains_dev.json [hunk 1]
```diff
@@ -726,6 +726,45 @@
                     "existentialDeposit": "100000000000000000",
                     "transfersEnabled": true
                 }
+            },
+            {
+                "assetId": 24,
+                "symbol": "GENS",
```
Comment (stepanLav) [inline chains/v3/chains_dev.json:732]: Should we add the GENS price @antonkhvorov ?

https://www.coingecko.com/ru/%D0%9A%D1%80%D0%B8%D0%BF%D1%82%D0%BE%D0%B2%D0%B0%D0%BB%D1%8E%D1%82%D1%8B/genshiro

File: chains/v3/chains_dev.json [hunk 2]
```diff
@@ -726,6 +726,45 @@
...
+                "typeExtras": {
+                    "currencyIdScale": "0x050f00",
+                    "currencyIdType": "acala_primitives.currency.CurrencyId",
+                    "existentialDeposit": "10000000000",
+                    "transfersEnabled": true
+                }
+            },
+            {
+                "assetId": 26,
+                "symbol": "CRAB",
```
Comment (stepanLav) [inline chains/v3/chains_dev.json:758]: Also looks like we have a price:
https://www.coingecko.com/ru/%D0%9A%D1%80%D0%B8%D0%BF%D1%82%D0%BE%D0%B2%D0%B0%D0%BB%D1%8E%D1%82%D1%8B/darwinia-crab-network-native-token

File: chains/v3/chains_dev.json (hunk 1 above)
Comment (antonkhvorov) [reply chains/v3/chains_dev.json:732]: Yes, I think it would be good

File: chains/v3/chains_dev.json [hunk 3]
```diff
@@ -1127,6 +1166,59 @@
...
+                    "currencyIdType": "node_primitives.currency.CurrencyId",
+                    "existentialDeposit": "1000",
+                    "transfersEnabled": true
+                }
+            },
+            {
+                "assetId": 8,
+                "symbol": "MOVR",
+                "precision": 18,
+                "type": "orml",
```
Comment (stepanLav) [inline chains/v3/chains_dev.json:1188]: "priceId": "moonriver",

File: chains/v3/chains_dev.json [hunk 4]
```diff
@@ -1127,6 +1166,59 @@
...
+                    "currencyIdType": "node_primitives.currency.CurrencyId",
+                    "existentialDeposit": "1000000000000",
+                    "transfersEnabled": true
+                }
+            },
+            {
+                "assetId": 9,
+                "symbol": "PHA",
+                "precision": 12,
+                "type": "orml",
```
Comment (stepanLav) [inline chains/v3/chains_dev.json:1201]:
```suggestion
                "type": "orml",
                "priceId": "pha",
```

File: chains/v3/chains_dev.json (hunk 1 above)
Comment (leohar) [reply chains/v3/chains_dev.json:732]: done

**Review by stepanLav — APPROVED** [review]

File: chains/v3/chains_dev.json (hunk 2 above)
Comment (leohar) [reply chains/v3/chains_dev.json:759]: done, also added to network token

File: chains/v3/chains_dev.json (hunk 4 above)
Comment (leohar) [reply chains/v3/chains_dev.json:1204]: done

**Review by ERussel — APPROVED** [review]

**Pull request merged**